netbout-web/pom.xml

```diff
@@ -45,6 +45,11 @@
     </properties>
     <dependencies>
         <dependency>
+            <groupId>commons-codec</groupId>
+            <artifactId>commons-codec</artifactId>
+            <version>1.10</version>
```

Comment:
@antonini probably version number is not required, jcabi parent already has version included (in dependencyManagement section), so please remove this version line